games/c.yaml

```diff
@@ -230,6 +230,7 @@
   - TypeScript
   license:
   - MIT
+content: commercial
```

Comment:
```suggestion
  content: commercial
```
oops; I think the indentation is wrong

Reply:
It should be ok in PR itself, I fixed it